apps/atomy-q/API/app/Http/Controllers/Api/V1/RequisitionVendorSelectionController.php

```
@@ -0,0 +1,205 @@
+                    'rfq_id' => $rfq->id,
+                    'vendor_id' => $vendorId,
+                    'selected_by_user_id' => $this->userId($request),
+                    'selected_at' => now(),
+                ]);
+
+                $selection->setRelation('vendor', $vendors->get($vendorId));
+                $selections->push($selection);
+            }
+
+            return $selections;
+        });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'update'. (undefined)

(StaticAccess)